content/en/docs/pipelines/sdk/python-function-components.md

```diff
@@ -133,11 +139,11 @@ def add_pipeline(
   # `task.outputs['output_name']`.
   second_add_task = add_op(first_add_task.output, b)
     
-# Specify argument values for your pipeline run.
-arguments = {'a': '7', 'b': '8'}
+  # Specify argument values for your pipeline run.
+  arguments = {'a': '7', 'b': '8'}
```

Comment:
no indentation here. This is meant to be outside the pipeline function (`calc_pipeline`) definition

Reply:
Yikes! Thanks for pointing that out!